webpack/assets/javascripts/foreman_tools.js

```diff
@@ -63,6 +63,7 @@ export function activateTooltips(elParam = 'body') {
   el.tooltip({
     selector: '[rel="twipsy"],*[title]:not(*[rel],.fa,.pficon)',
     container: 'body',
+    trigger: 'hover',
```

Comment:
The default for this is `'hover focus'`, however we don't actually use tooltips for places that require focus (we have popover for that).